src/main/java/seedu/duke/command/SearchModuleCommand.java

```diff
@@ -91,6 +98,15 @@ private void processParams() throws YamomException {
         toSearchLevel = processSearchLevel(level);
         // validate semester
         toSearchSemester = processSemester(semester);
+
```

Comment:
I think we are just checking that if the moduleCode is not null then it shouldn't be empty right. In which case no need to && check with moduleTitle.

Reply:
Also, can use "".equals(toSearchModuleCode) instead of Objects.equals

Reply:
Will fix

Reply:
> I think we are just checking that if the moduleCode is not null then it shouldn't be empty right. In which case no need to && check with moduleTitle.

good catch